### PR TITLE
Rename staticAnalysis.md -> static-analysis.md #7742

### LINKED ISSUE #7742: Rename staticAnalysis.md -> static-analysis.md
For consistency with the way the other documents are named. Also involves updating links to it in the other documentation files, if any.

### PR DESCRIPTION
**Pull request by lazy-sunshine**
Fixes #7742

### CODE REVIEW

File: static-analysis.md [hunk 1]
```diff
@@ -180,7 +179,7 @@ and [set up project specific settings and dependencies](setting-up.md#step-3-set
 1. Click `OK`.
 1. Copy `$PROJECT_DIR$/static-analysis/teammates-stylelint.yml` to `$PROJECT_DIR$/.stylelintrc.yml`.
 
-### blanket.js
+### blanket.js
```
Comment (Mynk96) [inline static-analysis.md]: remove this additional white space.

**Comment by lazy-sunshine** [comment]
I have removed the white space.Please merge :)

File: static-analysis.md [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (Mynk96) [inline static-analysis.md]: Nice work, almost there. This change seems redundant. Please revert.

**Comment by whipermr5** [comment]
@lazy-sunshine May we have your full name for inclusion into the contributors' list? :)

**Comment by lazy-sunshine** [comment]
Hi,
My full name is Kirti Chaturved.
Thanks,
Kirti

**Pull request merged**